TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
I don't know what you had previously here but I would expect these to target `1.0.0-servicing-*`